packages/eslint-plugin/src/rules/unbound-method.ts

```diff
@@ -18,6 +18,61 @@ export type Options = [Config];
 
 export type MessageIds = 'unbound';
 
+const nativelyBoundMembers = [
+  ...[
+    Promise,
+    //    BigInt, // todo: enable when we drop node@8
+    Number,
+    Object,
+    String,
+    RegExp,
+    Symbol,
+    Array,
+    Proxy,
+    Date,
+  ].map(x => [x.name, x]),
+  ['Infinity', Infinity],
+  ['Atomics', Atomics],
+  ['Reflect', Reflect],
+  ['console', console],
+  ['Math', Math],
+  ['JSON', JSON],
+  ['Intl', Intl],
+]
+  .map(([namespace, object]) =>
+    Object.getOwnPropertyNames(object)
+      .filter(
+        name =>
+          !name.startsWith('_') &&
+          typeof (object as Record<string, unknown>)[name] === 'function',
+      )
+      .map(name => `${namespace}.${name}`),
+  )
+  .reduce((arr, names) => arr.concat(names), []);
+
+const isMemberImported = (
```

Comment:
This actually checks the opposite.
```suggestion
const isMemberNotImported = (
```

Reply:
I am officially out of words for this function - it has won.